docfiles/footer.html

```diff
@@ -20,12 +20,12 @@
         </div>
         <div class="ui section divider"></div>
         <div class="ui container horizontal small divided link list">
+            <a id="langpicker" tabindex="0" class="ui item" rel="noopener">
+                <i class="icon xicon globe" aria-hidden="true" role="presentation"></i>Language
+            </a>
             <a class="item" href="https://makecode.com/privacy" target="_blank" rel="noopener">Privacy &amp; Cookies</a>
             <a class="item" href="https://makecode.com/termsofuse" target="_blank" rel="noopener">Terms Of Use</a>
             <a class="item" href="https://makecode.com/trademarks" target="_blank" rel="noopener">Trademarks</a>
-            <!-- @ifdef crowdinproject -->
-            <a id="translatebtn" tabindex="0" class="item">Translate this page</a>
```

Comment:
please keep this. putting this at the bottom of the language picker is burying too much

Reply:
We talked about this in the design meeting.  Consensus language picker is good place to put this in. Consistent with the editor. If you are translating you will click on the language selection to see.